modules/web/src/app/cluster/details/cluster/edit-provider-settings/alibaba-provider-settings/component.ts

```diff
@@ -33,7 +33,10 @@ export class AlibabaProviderSettingsComponent implements OnInit, OnDestroy {
   readonly Control = Control;
   form: FormGroup;
 
-  constructor(private readonly _clusterService: ClusterService, private readonly _builder: FormBuilder) {}
+  constructor(
+    private readonly _clusterService: ClusterService,
+    private readonly _builder: FormBuilder
+  ) {}
```

Comment:
This change (and other similar changes) is due to an update in prettier 3.0 and unfortunately there is no option to disable it as of now. Ref: https://github.com/prettier/prettier/issues/5666#issuecomment-1642084041